values/coturn/prod-values.example.yaml

```diff
@@ -0,0 +1,4 @@
+secrets:
+  zrestSecrets:
+    - ""
+  zrestSecret: ""
```

Comment:
Are we sure this is needed? I couldn't find a reference in the `coturn` chart.

**suggestion:** delete.
```suggestion
```